dataeng/resources/org-exporter-worker.sh

```diff
@@ -2,18 +2,11 @@ TODAY=$(date +%d)
 
 env | sort
 
-# Uncomment the following section only while working DENG-382
-#
-# Fetch the latest config:
-#mkdir ${WORKSPACE}/remote-config
-#aws s3 cp s3://edx-remote-config/prod/lms.yml ${WORKSPACE}/remote-config/lms.yml
-#aws s3 cp s3://edx-remote-config/prod/studio.yml ${WORKSPACE}/remote-config/studio.yml
```

Comment:
Jokes on me for thinking the solution was this simple.  It was complicated enough that I ended up having to break it out to a separate file (remote-config.sh).